From: Dan Kilman <dankilman@gmail.com>
Date: Sun, 13 Apr 2014 18:06:13 +0300
Subject: bash completion: fix __get_startable_units

(cherry picked from commit a163b64c4b08e8a4ad39a9a295acf3d1634024a3)
(cherry picked from commit cc9db03683d794e2894bb1a682724fb3493e0bf5)
---
 shell-completion/bash/systemctl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/shell-completion/bash/systemctl b/shell-completion/bash/systemctl
index 857552d..1e4d07a 100644
--- a/shell-completion/bash/systemctl
+++ b/shell-completion/bash/systemctl
@@ -56,7 +56,7 @@ __get_all_units      () { __systemctl $1 list-units --all \
 __get_active_units   () { __systemctl $1 list-units       \
         | { while read -r a b; do echo " $a"; done; }; }
 __get_startable_units () { __systemctl $1 list-units --all -t service,timer,socket,mount,automount,path,snapshot,swap \
-        | { while read -r a b c d; do [[ $c == "inactive" || $c == "failed " ]] && echo " $a"; done; }; }
+        | { while read -r a b c d; do [[ $c == "inactive" || $c == "failed" ]] && echo " $a"; done; }; }
 __get_failed_units   () { __systemctl $1 list-units       \
         | { while read -r a b c d; do [[ $c == "failed"   ]] && echo " $a"; done; }; }
 __get_enabled_units  () { __systemctl $1 list-unit-files  \
